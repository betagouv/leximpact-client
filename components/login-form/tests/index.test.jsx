/* eslint
    indent: [2, 2],
    semi: [2, "always"],
    react/jsx-indent: [2, 2],
    react/jsx-indent-props: [2, 2]
    max-nested-callbacks: [2, { "max": 4 }]
*/
import { createShallow } from "@material-ui/core/test-utils";

import LoginForm from "../index";

describe("components | login-form | index", () => {
  let shallow;
  beforeAll(() => {
    shallow = createShallow();
  });

  describe("snapshot", () => {
    it("doit correspondre avec les props requises", () => {
      const props = {};
      const wrapper = shallow(<LoginForm {...props} />);
      expect(wrapper).toBeDefined();
      expect(wrapper).toMatchSnapshot();
    });
  });
});
