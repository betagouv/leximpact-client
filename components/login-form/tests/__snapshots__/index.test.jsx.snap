// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | login-form | index snapshot doit correspondre avec les props requises 1`] = `
<RequestTokenForm
  classes={
    Object {
      "avatar": "RequestTokenForm-avatar-2",
      "avatarIcon": "RequestTokenForm-avatarIcon-3",
      "container": "RequestTokenForm-container-1",
      "form": "RequestTokenForm-form-4",
      "spanBolder": "RequestTokenForm-spanBolder-7",
      "spanLighter": "RequestTokenForm-spanLighter-6",
      "submit": "RequestTokenForm-submit-5",
    }
  }
  initialValues={
    Object {
      "domains": Array [
        "@assemblee-nationale.fr",
      ],
      "role": "depute",
      "username": null,
    }
  }
/>
`;
